assets/webpack/webpack.prod.js

```diff
@@ -15,9 +15,6 @@ module.exports = (_, argv) => {
       warnings:     true,
       version:      true,
     },
-    optimization: {
-      splitChunks: { chunks: 'all' },
-    },
```

Comment:
This is the main culprit for why `$` couldn't be found. I'm entirely sure, but I think it causes naming issues and the compiled html that phoenix produces doesn't point to the right chunk.